packages/fxa-auth-server/lib/payments/paypal/helper.ts

```diff
@@ -677,4 +677,55 @@ export class PayPalHelper {
 
     return;
   }
+
+  /**
+   * Attempts to refund all of the invoices passed, provided they're created via PayPal
+   * This will invisibly do nothing if the invoice is not billed through PayPal, so be mindful
+   * if using it elsewhere and need confirmation of a refund.
+   */
+  public async refundInvoice(invoice: Stripe.Invoice) {
```

Comment:
Could we add some tests for refundInvoice?

The logic here looks mostly similar to `PayPalHelper.refundInvoices`. Would it make sense to update `refundInvoices` so that it uses `refundInvoice`? Might make testing easier as well.